examples/nlp/machine_translation/conf/nmt_megatron_infer.yaml

```diff
@@ -13,4 +13,10 @@ source_lang: null
 target_lang: null
 tensor_model_parallel_size: 1
 pipeline_model_parallel_size: 1
-pipeline_model_parallel_split_rank: 0
\ No newline at end of file
+pipeline_model_parallel_split_rank: 0
+
+# adapter information
+model:
```

Comment:
Why do you need these args here? They should already be in the config of the model you are restoring right?

Reply:
Thanks for noticing this, I updated and removed unnecessary args